docs/migration_guide.md

```diff
@@ -0,0 +1,92 @@
+
+2. Install [curl](https://everything.curl.dev/get){:target="_blank"}.
+
+3. Install [jq](https://jqlang.github.io/jq/){:target="_blank"}.
+
+4. Set up a publicly accessible Kubernetes cluster. 
+
+    Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a publicly accessible Kubernetes cluster.
+
+    We recommend setting up Percona Everest on the Amazon Elastic Kubernetes Service (EKS) or Google Kubernetes Engine (GKE), as Percona Everest may not work as expected on local Kubernetes installations (minikube, kind, k3d, or similar products) due to network issues.
+
+   
+    [Create EKS cluster :material-arrow-right:](quickstart-guide/eks.md){.md-button} [Create GKE cluster :material-arrow-right:](quickstart-guide/gke.md){.md-button}
+
+5. Verify that you have access to the Kubernetes cluster that you want to use with Percona Everest. By default, Percona Everest uses the kubeconfig file available under `~/.kube/config`. 
+
+    To verify access to the Kubernetes cluster, run:
+   
+    ```sh 
+    kubectl get nodes
+    ```
+
+??? example "Expected output"
+    
+    ```{.text .no-copy}
+    NAME                                    STATUS   ROLES    AGE   VERSION
+    gke-<name>-default-pool-75d48bfc-bx8g   Ready    <none>   11h   v1.26.7-gke.500
+    gke-<name>-default-pool-75d48bfc-c2df   Ready    <none>   11h   v1.26.7-gke.500
+    gke-<name>-default-pool-75d48bfc-zl7k   Ready    <none>   11h   v1.26.7-gke.500
+    ```
+
+
+## Migration procedure
+
+1. Create a new Kubernetes cluster.
+2. [Provision](use/db_provision.md) the cluster.
+3. Migrate backup storages from PMM to Percona Everest.
+
+    To transfer backup storage from PMM to Percona Everest, follow these steps:
+
+    - Log in to the PMM UI and navigate to the **Backup Storages** section.
+    - Take a [backup](https://docs.percona.com/percona-monitoring-and-management/get-started/backup/index.html).
+    - Log in to Percona Everest and go to the **[Backup Storages](use/backupRestore.md)** section on the UI.
+    - Copy the backup from PMM and paste it to Percona Everest.
+
+4. Migrate secrets for database clusters from one Kubernetes cluster to a new cluster using the pattern `everest-secrets-dbclusterName`. 
+   
+    !!! note alert alert-primary "Note"
+        Keep names consistent across two Kubernetes clusters.
+
+5. Create a new database cluster using the webUI. Run through the creation wizard to select resources, set the name, and configure the backup storage and monitoring of the cluster.
+6. Restore data for your database clusters using the operator's backup and restore features.
+
+    !!! note alert alert-primary "Note"
+        - The restoration won't be available for you in the web UI for this run, but other backups/restores will be there.
+
+            1. For PXC https://docs.percona.com/percona-operator-for-mysql/pxc/backups.html
```

Comment:
This list shouldn't be numbered, these are not steps, but options. Can we add bullets?

Reply:
fixed.